config/flags.go

```diff
@@ -49,3 +49,9 @@ func readTracingConfig(cfg TracingConfig, fs *flag.FlagSet) {
 	fs.StringVar(&cfg.CertFile, TracingCertFileKey, "", "The tracing certificate file")
 	fs.StringVar(&cfg.KeyFile, TracingKeyFileKey, "", "The tracing key file")
 }
+
+func readDBConfig(cfg DBConfig, fs *flag.FlagSet) {
+	fs.StringVar(&cfg.DBName, DBNameKey, "", "database name")
+	fs.StringVar(&cfg.DBPath, DBPathKey, "", "path to database")
+	fs.StringVar(&cfg.MigrationsPath, MigrationsPathKey, "", "path to migration scripts")
+}
```

Comment:
As these don't have default values, should we include the flags in the example config files?